### PR TITLE
[GH-15654] Introduce MLFlow flavor for working with mojos and pojos

### PR DESCRIPTION
**Pull request by mn-mikke**
Closes GH-15654

### CODE REVIEW

File: h2o-py-mlflow-flavor/.gitignore [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Backup files
```
Comment (valenad1) [inline h2o-py-mlflow-flavor/.gitignore:1]: Do we need another .gitignore? A lot of staff is covered in h2o-3/.gitignore

**Review by krasinski — DISMISSED** [review]
simply awesome @mn-mikke :muscle: I really like the docs

File: h2o-py-mlflow-flavor/.gitignore (hunk 1 above)
Comment (mn-mikke) [reply h2o-py-mlflow-flavor/.gitignore]: I removed the file, good catch

**Comment by valenad1** [comment]
Question: 

How we will declare compatibility with h2o-3? Is it going to be same as h2o clients - equal, it means that with each release we release also ml-flow? Or the mlflow flavor will be compatible with future version unless we say otherwise. What is the lowest version we declare to support - 3.46.0.1?

**Review by valenad1 — APPROVED** [review]
Good change, thank you!

**Comment by valenad1** [comment]
> Question:
> 
> How we will declare compatibility with h2o-3? Is it going to be same as h2o clients - equal, it means that with each release we release also ml-flow? Or the mlflow flavor will be compatible with future version unless we say otherwise. What is the lowest version we declare to support - 3.46.0.1?
> 
> The easiest would be to match h2o-3 version IMHO.

Its separate from H2O-3, it uses only basic API and should be compatible across previous and future versions.

**Pull request merged**